dev/com.ibm.websphere.security.audit/src/com/ibm/ws/security/audit/utils/package-info.java

```diff
@@ -0,0 +1,16 @@
+/*******************************************************************************
+ * Copyright (c) 2011 IBM Corporation and others.
```

Comment:
New file? 2018?

Reply:
Originally, I tried to split out some audit specfics from the com.ibm.websphere.security project into a com.ibm.ws.websphere.security.audit package, but then reverted that.   So com.ibm.websphere.security.audit no longer exists.